profiling/ldms-kokkos-sampler/Makefile

```diff
@@ -0,0 +1,32 @@
+
+
+OVIS_DIR=/projects/ovis/streams/opt/ovis
+
+CXX=g++
+CXXFLAGS=-O3 -std=c++11 -g \
+	-I$(OVIS_DIR)/include/ -I./include
+SHARED_CXXFLAGS=-shared -fPIC
+LDFLAGS=-L$(OVIS_DIR)/lib
+LIBS=-lldmsd_stream -lldms -lrt
+
+all: kp_kernel_ldms.so
+
+MAKEFILE_PATH := $(subst Makefile,,$(abspath $(lastword $(MAKEFILE_LIST))))
+
+CXXFLAGS+=-I${MAKEFILE_PATH}
+
+kp_kernel_ldms.so: ${MAKEFILE_PATH}kp_kernel_ldms.cpp ${MAKEFILE_PATH}kp_kernel_info.h
```

Comment:
We only have the `kp_sampler.so` code in this PR so we need to remove references to the LDMS tools.